### DISCUSSION TITLE
Detailed logging

### DISCUSSION THREAD

**Discussion by skovalevskii** [XTLS/Xray-core#2566] [Q&A] [answered]
Hi guys. Do you know any other ways to get more detailed access logs?
I use access info logs but it's not detailed enough. All I can see is something like "Google.com" and thats it without any additional info like "google.com/main_page" etc

Thanks

**Comment by FranzKafkaYu** [comment]
Just adjust your log level for more detailed logs in config.json

**Comment by flowerinsnowdh** [comment, marked as answer]
显示为 `accepted //www.google.com:443` 的时候，大概率这个连接是 HTTPS 连接，也就是说，`/main_page` 这一段是加密的。在 TLS 的加密下，即使是 Xray 也无法捕获访问的路径

当访问不通过 TLS 加密时，配置了 `sniffing` 的入站可以捕获到路径，像这样

`accepted http://149.154.175.54:80/api`

<hr />

When logged as `accepted //www.google.com:443`, there is a high probability that the connection is an HTTPS connection, that is to say, the path `/main_page` is encrypted. Under TLS encryption, even Xray cannot capture the accessed path

When access is not encrypted via TLS, inbound configured with `sniffing` can capture the path, like this

`accepted http://149.154.175.54:80/api`

**Comment by chika0801** [comment]
Xray's logging level, if you use debug. doesn't do what you're looking for either. the logging inside Xray has never provided the ability to log all the URLs in such detail as you're describing.
